static/app/views/insights/mobile/screens/components/vitalDetailPanel.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ExternalLink from 'sentry/components/links/externalLink';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {PageAlert, PageAlertProvider} from 'sentry/utils/performance/contexts/pageAlert';
+import {PERFORMANCE_SCORE_COLORS} from 'sentry/views/insights/browser/webVitals/utils/performanceScoreColors';
+import DetailPanel from 'sentry/views/insights/common/components/detailPanel';
+import useCrossPlatformProject from 'sentry/views/insights/mobile/common/queries/useCrossPlatformProject';
+import {
+  PerformanceScore,
+  type VitalItem,
+  type VitalStatus,
+} from 'sentry/views/insights/mobile/screens/utils';
+
+export function VitalDetailPanel({
+  vital,
+  status,
+  onClose,
+}: {
+  onClose: () => void;
+  status: VitalStatus | undefined;
+  vital: VitalItem | undefined;
+}) {
+  const {selectedPlatform} = useCrossPlatformProject();
+
+  const platformDocsLink = vital?.platformDocLinks[selectedPlatform];
+  const sdkDocsLink = vital?.sdkDocLinks[selectedPlatform];
+
+  return (
+    <PageAlertProvider>
+      <DetailPanel
+        detailKey={vital !== undefined ? vital.field : undefined}
```

Comment:
Tiny nit, feel free to ignore this or put it in another PR!

```suggestion
        detailKey={vital?.field}
```